projects/ad2s1210_iio/app/ad2s1210_user_config.c

```diff
@@ -0,0 +1,65 @@
+/***************************************************************************//**
+ * @file    ad2s1210_user_config.c
+ * @brief   User configurations for AD2S1210 No-OS driver
+********************************************************************************
+* Copyright (c) 2023 Analog Devices, Inc.
+* Copyright (c) 2023 BayLibre, SAS.
+* All rights reserved.
+*
+* This software is proprietary to Analog Devices, Inc. and its licensors.
+* By using this software you agree to the terms of the associated
+* Analog Devices Software License Agreement.
+*******************************************************************************/
+
+/******************************************************************************/
+/***************************** Include Files **********************************/
+/******************************************************************************/
+
+#include "ad2s1210_user_config.h"
+#include "app_config.h"
```

Comment:
Do we need this include here?

Reply:
yes for AD2S1210_MAX_CLKIN, and RESOLUTION if we will add it in app_config.h (suggestion above)